solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -182,6 +157,37 @@ private ClusterState fetchClusterState(
     return cs;
   }
 
+  private DocCollection getDocCollectionFromObjects(Map.Entry<String, Object> e, int zNodeVersion) {
+    @SuppressWarnings("rawtypes")
+    Map m = (Map) e.getValue();
+    Long creationTimeMillisFromClusterStatus = (Long) m.get("creationTimeMillis");
```

Comment:
wow that's a long var name.  I think the `FromClusterStatus` suffix isn't needed; we aren't disambiguating multiple sources of creation time.

Reply:
Updated